plugins/pom.xml

```diff
@@ -59,6 +59,7 @@
         <module>api/solidfire-intg-test</module>
 
         <module>backup/dummy</module>
+        <module>backup/networker</module>
```

Comment:
@fermosan Does this only support KVM or all hypervisors?

Reply:
For starters only KVM. It could be ported for VMWARE (with some modifications for supporting vsphere). If DELL plans to support clients for other we can port it.